### PR TITLE
Various updates for Moriond

### PR DESCRIPTION
**Pull request by blinkseb**
See each commit for more details.

### CODE REVIEW

**Comment by swertz** [comment]
Great! :+1: From a quick look, it seems cMVAv2 medium scale factors for light jets are missing?

**Comment by blinkseb** [comment]
No, in fact, they have not changed since the last update, I was surprised too :)

File: README.md [hunk 1]
```diff
@@ -52,7 +59,9 @@ cd ${CMSSW_BASE}/external/${SCRAM_ARCH}
 git clone https://github.com/ikrav/RecoEgamma-ElectronIdentification.git data/RecoEgamma/ElectronIdentification/data
 cd data/RecoEgamma/ElectronIdentification/data
 git checkout egm_id_80X_v1
-cd ${CMSSW_BASE}/src
+
+cd ${CMSSW_BASE}/src/EgammaAnalysis/ElectronTools/data
+git clone https://github.com/ECALELFS/ScalesSmearings.git
```
Comment (OlivierBondu) [inline README.md:64]: is it expected to clone the repo twice in the same directory ?

File: bootstrap_jenkins.sh [hunk 2]
```diff
@@ -5,6 +5,7 @@
 
 git cms-merge-topic ikrav:egm_id_80X_v2
 git cms-merge-topic rafaellopesdesa:Regression80XEgammaAnalysis_v2
+git cms-merge-topic shervin86:Moriond2017_JEC_energyScales
 git cms-merge-topic gpetruc:badMuonFilters_80X_v2
```
Comment (OlivierBondu) [inline bootstrap_jenkins.sh:9]: seems the 
```
cd ${CMSSW_BASE}/src/EgammaAnalysis/ElectronTools/data
 git clone https://github.com/ECALELFS/ScalesSmearings.git
```
is not in the bootstrap, is this expected ?

Comment (OlivierBondu) [reply bootstrap_jenkins.sh:9]: nevermind, it's in the `postbuild.sh`

File: README.md (hunk 1 above)
Comment (blinkseb) [reply README.md:64]: Nop :)

**Pull request merged**